--- a/relay-rust/src/main.rs
+++ b/relay-rust/src/main.rs.paths
@@ -299,7 +299,7 @@ impl Command for RelayCommand {
 
 fn cmd_install(serial: Option<&String>) -> Result<(), CommandExecutionError> {
     info!(target: TAG, "Installing gnirehtet client...");
-    exec_adb(serial, vec!["install", "-r", "gnirehtet.apk"])
+    exec_adb(serial, vec!["install", "-r", "/usr/share/gnirehtet/gnirehtet.apk"])
 }
 
 fn cmd_uninstall(serial: Option<&String>) -> Result<(), CommandExecutionError> {
